src/assets/i18n/fr.ts

```diff
@@ -1,528 +1,528 @@
+        "loadingLastestComment":"Chargement des derniers commentaires",
+        "showless":"montrer moins",
+        "showmore":"montrer plus",
+        "didnotrelease":"Feeds peut fonctionner normalement après la publication de DID sur la chaîne",
+        "unknown":"Inconnu",
+        "unfollow":"Se désabonner",
+        "adding":"Ajouter",
+        "feedsAlreadyAdded":"Ce Feed a déjà été ajouté",
+        "notFollowYet":"Pas encore suivi"
     },
     "app":{
-        "profileDetails":"Profile Details",
-        "feedSources":"Publisher Account",
-        "about":"About Feeds",
+        "profileDetails":"Details du Profil",
+        "feedSources":"Compte Editeur",
+        "about":"A propos de Feeds",
         "donation":"Donation",
-        "signout":"Sign out",
-        "des":"Sign out successfully",
-        "settings":"Settings"
+        "signout":"Se déconnecter",
+        "des":"Déconnection réussie",
+        "settings":"Réglages"
     },
     "SigninPage":{
-        "signIn":"Sign In to Feeds",
-        "Pleasewait":"Please wait...",
+        "signIn":"Se connecter à Feeds",
+        "Pleasewait":"Veuillez patienter...",
         "feeds": 'Feeds',
-        "des":"Where meaningful",
-        "des1":"conversations happens",
-        "des2":"Learn more about Feeds?",
-        "des3":"What is Feeds?",
-        "des4":"Follow your interests.",
-        "des5":"Hear what people are talking about.",
-        "des6":"Join the conversation.",
+        "des":"Où des conversations",           // Grammar wise, it should read "happen" not "happens" 
+        "des1":"significatives ont lieu",
+        "des2":"En savoir plus sur Feeds?",
+        "des3":"Qu'est-ce qu'est Feeds?",
+        "des4":"Suivez vos centres d'intérêt.",
+        "des5":"Écoutez ce dont les gens parlent.",
+        "des6":"Participer à la conversation.",  //change "join" by "participate"
     },
     "FeedsPage":{
-        "tabTitle1":"My Feeds Timeline",
-        "tabTitle2":"My Profile",
+        "tabTitle1":"Mon historique Feeds",
+        "tabTitle2":"Mon Profil",
         "tabTitle3":"Notifications",
-        "tabTitle4":"Explore Feeds"
+        "tabTitle4":"Explorer Feeds"
     },
     "HomePage":{
-        "allempty":"All Empty",
-        "emptyDes":"Not following any Feeds yet",
-        "exploreFeeds":"Explore Feeds",
-        "minutesAgo":" minutes ago",
-        "hoursAgo":" hours ago",
-        "daysAgo":" days ago",
-        "onedayAgo":" day ago",
-        "onehourAgo":" hour ago",
-        "oneminuteAgo":" minute ago"
+        "allempty":"Tout vide",
+        "emptyDes":"Pas de Feeds suivi pour l'instant",
+        "exploreFeeds":"Explorer Feeds",
+       // "minutesAgo":" ", 
+       // "hoursAgo":" hours ago",
+       // "daysAgo":" days ago",
+       // "onedayAgo":" day ago",              //"5 min ago" = "Il y a 5 min", you can't translate this on this format
```

Comment:
@songjiazhun The translation string should be modified and include a variable in the string.
Something like:
"onedayago": "{{daysAgo}} day ago"

instead of:
1 {{ 'onedayago' | translate }}